TABSwiftLayout/Constraint.swift

```diff
@@ -251,11 +262,6 @@ public struct Constraint: ConstraintDefinition {
     self.priority = 250
```

Comment:
It could, but because this lib supports iOS and OSX its simpler to define a value rather than mapping all the NS/UILayoutPriority... options. Unless you all prefer that ;)